meta/runtime.yml

```diff
@@ -0,0 +1,2 @@
+---
+requires_ansible: ">=2.9"
```

Comment:
2.11 for the playbook feature: https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#using-a-playbook-from-a-collection

Reply:
I'll fix it. But let's see a sucessful build first.

Reply:
Changed the requirement to ansible-core >= 2.11